lib/stack_master/parameter_resolvers/env.rb

```diff
@@ -0,0 +1,17 @@
+module StackMaster
+  module ParameterResolvers
+    class Env < Resolver
+      array_resolver
+
+      def initialize(config, stack_definition)
+        @config = config
+        @stack_definition = stack_definition
+      end
+
+      def resolve(value)
+        ENV[value]
```

Comment:
Would it be appropriate to raise an error if the expected env var isn't present?